docs/source/en/basics/colossalai_booster.md

```diff
@@ -0,0 +1,124 @@
+# colossal-ai booster
+
+**Prerequisite:**
+- [Distributed Training](../concepts/distributed_training.md)
+- [Colossal-AI Overview](../concepts/colossalai_overview.md)
+
+## Introduction
+In our new design, `colossalai.booster` replaces the role of `colossalai.initialize` to inject features into your training components (e.g. model, optimizer, dataloader) seamlessly. With these new APIs, user can integrate their model with our parallelism features more friendly. Also calling `colossalai.booster` is the standard procedure before you run into your training loops. In the sections below, I will cover how `colossalai.booster` works and what we should take note of.
+
+### Plugin
+<p>Plugin is an important component that manages parallel configuration (eg: The gemini plugin encapsulates the gemini acceleration solution). Currently supported plugins are as follows:</p>
+
+***GeminiPlugin:*** <p> This plugin wrapps the Gemini acceleration solution, that ZeRO with chunk-based memory management. </p>
+
+***TorchDDPPlugin:*** <p>This plugin wrapps the DDP acceleration solution, it implements data parallelism at the module level which can run across multiple machines. </p>
+
+***LowLevelZeroPlugin:*** <p>This plugin wraps the 1/2 stage of Zero Redundancy Optimizer. Stage 1 : Shards optimizer states across data parallel workers/GPUs. Stage 2 : Shards optimizer states + gradients across data parallel workers/GPUs.</p>
+
+### API of booster
+Booster.__init__(...):
```

Comment:
You can use API reference here like https://github.com/hpcaitech/ColossalAI/blob/b37797ed3d3d6af294a095397b4bc135264b8c6a/docs/source/en/features/nvme_offload.md?plain=1#L258